spec/models/user_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'User model' do
+    before(:example) do
+      User.destroy_all
+      @user = User.create(first_name: 'Amigo', last_name: 'Soy',
+                          email: 'my@email.com', password: '321321')
+    end
+
+    it 'is invalid if names are empty' do
+      @user.first_name = nil
+      expect(@user).to_not be_valid
+      @user.first_name = 'Amigo'
+      @user.last_name = nil
+      expect(@user).to_not be_valid
+    end
+
```

Comment:
- [ ] Some of your tests are failing. Please fix this.
![Screenshot from 2022-05-24 15-21-18](https://user-images.githubusercontent.com/65860678/170046957-411a3e02-b661-40c2-b3ff-5c1617afc81a.png)

Reply:
Thank you! @uwadonat 

For some reason, the tests failed just on the first try. After repeating the `rspec` command, everything worked.

I edited so it runs correctly from the first try.